### PR TITLE
Adding WebJobsAuthenticationEvents prefix and removing unnecessary namespaces

### PR DESCRIPTION
**Pull request by HarmanDhunna**
Adding WebJobsAuthenticationEvents prefix and removing unnecessary namespaces. [API View](https://apiview.dev/Assemblies/Review/9e152c5745014a6d8a620a552d7a50ff) has been approved with the new names and reduction of namespaces

### CODE REVIEW

**Review by hakimms — APPROVED** [review]

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,37 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
-
-### Features Added
+## 1.0.0-beta.7 (2024-04-15)
 
 ### Breaking Changes
```
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md:5]: Beta releases do not guarantee API compatibility and we don't classify changes between beta versions as breaking.  In order to ensure that we have high visibility on stable breaking changes, please move these to "Other changes" and avoid the breaking changes section.

Comment (HarmanDhunna) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md]: Done.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,37 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.0-beta.7 (2024-04-15)
```
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.7 (2024-04-15)

```

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,37 @@
...
 
 ### Other Changes
+- Consildated namespaces. Removing:
```
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md:6]:
```suggestion

- Consildated namespaces. Removing:
```

**Review by jsquire — APPROVED** [review]

**Pull request merged**